internal/app/backendapi/apisrv/apisrv.go

```diff
@@ -87,16 +87,14 @@ func (s *backendAPI) CreateMatch(c context.Context, req *pb.CreateMatchRequest)
 	moID := xid.New().String()
 	requestKey := moID + "." + profile.Id
 
-	/*
-		// Debugging logs
-		beLog.Info("Pools nil? ", (profile.Pools == nil))
-		beLog.Info("Pools empty? ", (len(profile.Pools) == 0))
-		beLog.Info("Rosters nil? ", (profile.Rosters == nil))
-		beLog.Info("Rosters empty? ", (len(profile.Rosters) == 0))
-		beLog.Info("config set for json.pools?", s.cfg.IsSet("jsonkeys.pools"))
-		beLog.Info("contents key?", s.cfg.GetString("jsonkeys.pools"))
-		beLog.Info("contents exist?", gjson.Get(profile.Properties, s.cfg.GetString("jsonkeys.pools")).Exists())
-	*/
+	// Debugging logs
+	beLog.Trace("Pools nil? ", (profile.Pools == nil))
+	beLog.Trace("Pools empty? ", (len(profile.Pools) == 0))
+	beLog.Trace("Rosters nil? ", (profile.Rosters == nil))
+	beLog.Trace("Rosters empty? ", (len(profile.Rosters) == 0))
+	beLog.Trace("config set for json.pools?", s.cfg.IsSet("jsonkeys.pools"))
+	beLog.Trace("contents key?", s.cfg.GetString("jsonkeys.pools"))
+	beLog.Trace("contents exist?", gjson.Get(profile.Properties, s.cfg.GetString("jsonkeys.pools")).Exists())
```

Comment:
Would it make more sense to add the debugging logs for other functions if we wanna enable it here?

Reply:
If the code was commented out I'd just delete it. No one is going to miss it.

Reply:
For this, I'm not trying to change the logging, just re-prioritize the current logging to reduce spam.  We're going to be changing so much for 0.6 that I don't think spending time improving the quality of the logs is worth it.